examples/plot_crcns_dataset_example.py

```diff
@@ -0,0 +1,241 @@
+"""
+============================================
+CRCNS DataSet Example
+============================================
```

Comment:
I'm probably being a bit too OCDish, but just for the sake of consistency I'd keep as many === as the characters in the title. Like here: https://github.com/mne-tools/mne-python/blob/master/examples/preprocessing/plot_eog_artifact_histogram.py#L2-L4